File: Microservice/FrontEnd/FoodOrder/src/app/User/order.ts
import { OrderItem } from "./order-item";

export class Order {



      orderId!:number;
	
	
	 orderStatus!:string;
	
	
	 payamentStatus!:String;
	
	 orderAmount!:number;
	
    billingAddress!:string;
	
	 userId!:number;
	
	
	 orderItems:OrderItem[]=[];
	
}

File: Microservice/BackEnd/OrderIteam/OrderItem/src/main/java/com/example/Model/Order.java
package com.example.Model;

import java.util.Date;
import java.util.List;



import com.example.entity.OrderItem;

import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class Order {
private int orderId;
	
	
	private String orderStatus;
	
	
	private String payamentStatus;
	
	private double orderAmount;
	
	private String billingAddress;
	
	private int userId;
	
	private Date date;
	
	private List<OrderItem> orderItems;
	
}
